README.md

```diff
@@ -5,13 +5,17 @@ This repository hosts shared functionality for
 
 ![GitHub Issue Sync](https://github.com/paritytech/opstooling-js/actions/workflows/github-issue-sync.yml/badge.svg)
 
-# Installation
+## Docs
+
+![Docs deployed](https://github.com/paritytech/opstooling-js/actions/workflows/deploy-docs.yml/badge.svg)
 
-Choose a [tag](https://github.com/paritytech/opstooling-js/tags) and then run
+Find the docs in [paritytech.github.io/ostooling-js](https://paritytech.github.io/ostooling-js). They are deployed automatically.
```

Comment:
```suggestion
Find the docs in [paritytech.github.io/ostooling-js](https://paritytech.github.io/osptooling-js). They are deployed automatically.
```

Reply:
Good catch but it seems we both misstyped.

I forgot the p and you put it in the wrong place 🤣 

Gonna fix that

Reply:
ahaha, touché